x-pack/plugins/search_assistant/.i18nrc.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
We should remove this file, it doesn't do anything and we definitely don't want to explicitly reference a non-existent Japanese translations file